[2023-10-11T04:52:07.480+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: docker.virtualenv_python manual__2023-10-11T04:52:05.657777+00:00 [queued]>
[2023-10-11T04:52:07.485+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: docker.virtualenv_python manual__2023-10-11T04:52:05.657777+00:00 [queued]>
[2023-10-11T04:52:07.485+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-11T04:52:07.491+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): virtualenv_python> on 2023-10-11 04:52:05.657777+00:00
[2023-10-11T04:52:07.496+0000] {standard_task_runner.py:57} INFO - Started process 236 to run task
[2023-10-11T04:52:07.498+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'docker', 'virtualenv_python', 'manual__2023-10-11T04:52:05.657777+00:00', '--job-id', '1422', '--raw', '--subdir', 'DAGS_FOLDER/docker.py', '--cfg-path', '/tmp/tmpa8djvs8w']
[2023-10-11T04:52:07.498+0000] {standard_task_runner.py:85} INFO - Job 1422: Subtask virtualenv_python
[2023-10-11T04:52:07.524+0000] {task_command.py:415} INFO - Running <TaskInstance: docker.virtualenv_python manual__2023-10-11T04:52:05.657777+00:00 [running]> on host 806ac90253f5
[2023-10-11T04:52:07.562+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='docker' AIRFLOW_CTX_TASK_ID='virtualenv_python' AIRFLOW_CTX_EXECUTION_DATE='2023-10-11T04:52:05.657777+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-11T04:52:05.657777+00:00'
[2023-10-11T04:52:07.563+0000] {process_utils.py:182} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venvebjuz6_a
[2023-10-11T04:52:07.573+0000] {process_utils.py:186} INFO - Output:
[2023-10-11T04:52:08.358+0000] {process_utils.py:190} INFO - created virtual environment CPython3.8.18.final.0-64 in 269ms
[2023-10-11T04:52:08.358+0000] {process_utils.py:190} INFO -   creator CPython3Posix(dest=/tmp/venvebjuz6_a, clear=False, no_vcs_ignore=False, global=False)
[2023-10-11T04:52:08.358+0000] {process_utils.py:190} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/tmp/tmp1obsa3zx)
[2023-10-11T04:52:08.359+0000] {process_utils.py:190} INFO -     added seed packages: pip==23.2, setuptools==68.0.0, wheel==0.40.0
[2023-10-11T04:52:08.359+0000] {process_utils.py:190} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-10-11T04:52:08.382+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvebjuz6_a/bin/pip install -r /tmp/venvebjuz6_a/requirements.txt
[2023-10-11T04:52:08.392+0000] {process_utils.py:186} INFO - Output:
[2023-10-11T04:52:09.169+0000] {process_utils.py:190} INFO - WARNING: The directory '/home/***/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you should use sudo's -H flag.
[2023-10-11T04:52:09.396+0000] {process_utils.py:190} INFO - Collecting lightgbm==4.1.0 (from -r /tmp/venvebjuz6_a/requirements.txt (line 1))
[2023-10-11T04:52:09.396+0000] {process_utils.py:190} INFO -   Obtaining dependency information for lightgbm==4.1.0 from https://files.pythonhosted.org/packages/b8/9d/1ce80cee7c5ef60f2fcc7e9fa97f29f7a8de3dc5a08922b3b2f1e9106481/lightgbm-4.1.0-py3-none-manylinux_2_28_x86_64.whl.metadata
[2023-10-11T04:52:09.534+0000] {process_utils.py:190} INFO -   Downloading lightgbm-4.1.0-py3-none-manylinux_2_28_x86_64.whl.metadata (19 kB)
[2023-10-11T04:52:09.820+0000] {process_utils.py:190} INFO - Collecting pandas==2.0.3 (from -r /tmp/venvebjuz6_a/requirements.txt (line 2))
[2023-10-11T04:52:09.820+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pandas==2.0.3 from https://files.pythonhosted.org/packages/f8/7f/5b047effafbdd34e52c9e2d7e44f729a0655efafb22198c45cf692cdc157/pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-11T04:52:09.858+0000] {process_utils.py:190} INFO -   Downloading pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (18 kB)
[2023-10-11T04:52:10.002+0000] {process_utils.py:190} INFO - Collecting scikit-learn==1.3.1 (from -r /tmp/venvebjuz6_a/requirements.txt (line 3))
[2023-10-11T04:52:10.003+0000] {process_utils.py:190} INFO -   Obtaining dependency information for scikit-learn==1.3.1 from https://files.pythonhosted.org/packages/91/4e/752fd6e0b6b828a9cd815941fb70723eaef5d0f4efd8b9b0eee7864b3511/scikit_learn-1.3.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-11T04:52:10.041+0000] {process_utils.py:190} INFO -   Downloading scikit_learn-1.3.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (11 kB)
[2023-10-11T04:52:10.351+0000] {process_utils.py:190} INFO - Collecting numpy (from lightgbm==4.1.0->-r /tmp/venvebjuz6_a/requirements.txt (line 1))
[2023-10-11T04:52:10.352+0000] {process_utils.py:190} INFO -   Obtaining dependency information for numpy from https://files.pythonhosted.org/packages/98/5d/5738903efe0ecb73e51eb44feafba32bdba2081263d40c5043568ff60faf/numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata
[2023-10-11T04:52:10.390+0000] {process_utils.py:190} INFO -   Downloading numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.6 kB)
[2023-10-11T04:52:10.570+0000] {process_utils.py:190} INFO - Collecting scipy (from lightgbm==4.1.0->-r /tmp/venvebjuz6_a/requirements.txt (line 1))
[2023-10-11T04:52:10.607+0000] {process_utils.py:190} INFO -   Downloading scipy-1.10.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (34.5 MB)
[2023-10-11T04:52:14.182+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 34.5/34.5 MB 9.9 MB/s eta 0:00:00
[2023-10-11T04:52:14.344+0000] {process_utils.py:190} INFO - Collecting python-dateutil>=2.8.2 (from pandas==2.0.3->-r /tmp/venvebjuz6_a/requirements.txt (line 2))
[2023-10-11T04:52:14.383+0000] {process_utils.py:190} INFO -   Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
[2023-10-11T04:52:14.405+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 11.8 MB/s eta 0:00:00
[2023-10-11T04:52:14.490+0000] {process_utils.py:190} INFO - Collecting pytz>=2020.1 (from pandas==2.0.3->-r /tmp/venvebjuz6_a/requirements.txt (line 2))
[2023-10-11T04:52:14.491+0000] {process_utils.py:190} INFO -   Obtaining dependency information for pytz>=2020.1 from https://files.pythonhosted.org/packages/32/4d/aaf7eff5deb402fd9a24a1449a8119f00d74ae9c2efa79f8ef9994261fc2/pytz-2023.3.post1-py2.py3-none-any.whl.metadata
[2023-10-11T04:52:14.528+0000] {process_utils.py:190} INFO -   Downloading pytz-2023.3.post1-py2.py3-none-any.whl.metadata (22 kB)
[2023-10-11T04:52:14.584+0000] {process_utils.py:190} INFO - Collecting tzdata>=2022.1 (from pandas==2.0.3->-r /tmp/venvebjuz6_a/requirements.txt (line 2))
[2023-10-11T04:52:14.629+0000] {process_utils.py:190} INFO -   Downloading tzdata-2023.3-py2.py3-none-any.whl (341 kB)
[2023-10-11T04:52:14.659+0000] {process_utils.py:190} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 341.8/341.8 kB 12.0 MB/s eta 0:00:00
[2023-10-11T04:52:14.770+0000] {process_utils.py:190} INFO - Collecting joblib>=1.1.1 (from scikit-learn==1.3.1->-r /tmp/venvebjuz6_a/requirements.txt (line 3))
[2023-10-11T04:52:14.771+0000] {process_utils.py:190} INFO -   Obtaining dependency information for joblib>=1.1.1 from https://files.pythonhosted.org/packages/10/40/d551139c85db202f1f384ba8bcf96aca2f329440a844f924c8a0040b6d02/joblib-1.3.2-py3-none-any.whl.metadata
[2023-10-11T04:52:14.808+0000] {process_utils.py:190} INFO -   Downloading joblib-1.3.2-py3-none-any.whl.metadata (5.4 kB)
[2023-10-11T04:52:14.858+0000] {process_utils.py:190} INFO - Collecting threadpoolctl>=2.0.0 (from scikit-learn==1.3.1->-r /tmp/venvebjuz6_a/requirements.txt (line 3))
[2023-10-11T04:52:14.859+0000] {process_utils.py:190} INFO -   Obtaining dependency information for threadpoolctl>=2.0.0 from https://files.pythonhosted.org/packages/81/12/fd4dea011af9d69e1cad05c75f3f7202cdcbeac9b712eea58ca779a72865/threadpoolctl-3.2.0-py3-none-any.whl.metadata
[2023-10-11T04:52:14.895+0000] {process_utils.py:190} INFO -   Downloading threadpoolctl-3.2.0-py3-none-any.whl.metadata (10.0 kB)
[2023-10-11T04:52:14.952+0000] {process_utils.py:190} INFO - Collecting six>=1.5 (from python-dateutil>=2.8.2->pandas==2.0.3->-r /tmp/venvebjuz6_a/requirements.txt (line 2))
[2023-10-11T04:52:14.989+0000] {process_utils.py:190} INFO -   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[2023-10-11T04:52:15.067+0000] {process_utils.py:190} INFO - Downloading lightgbm-4.1.0-py3-none-manylinux_2_28_x86_64.whl (3.1 MB)
[2023-10-11T04:52:15.428+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.1/3.1 MB 8.5 MB/s eta 0:00:00
[2023-10-11T04:52:15.468+0000] {process_utils.py:190} INFO - Downloading pandas-2.0.3-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.4 MB)
[2023-10-11T04:52:17.576+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.4/12.4 MB 5.8 MB/s eta 0:00:00
[2023-10-11T04:52:17.629+0000] {process_utils.py:190} INFO - Downloading scikit_learn-1.3.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.1 MB)
[2023-10-11T04:52:19.294+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 11.1/11.1 MB 6.7 MB/s eta 0:00:00
[2023-10-11T04:52:19.333+0000] {process_utils.py:190} INFO - Downloading joblib-1.3.2-py3-none-any.whl (302 kB)
[2023-10-11T04:52:19.374+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 302.2/302.2 kB 7.5 MB/s eta 0:00:00
[2023-10-11T04:52:19.414+0000] {process_utils.py:190} INFO - Downloading numpy-1.24.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (17.3 MB)
[2023-10-11T04:52:21.643+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 17.3/17.3 MB 8.1 MB/s eta 0:00:00
[2023-10-11T04:52:21.681+0000] {process_utils.py:190} INFO - Downloading pytz-2023.3.post1-py2.py3-none-any.whl (502 kB)
[2023-10-11T04:52:21.736+0000] {process_utils.py:190} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 502.5/502.5 kB 9.3 MB/s eta 0:00:00
[2023-10-11T04:52:21.773+0000] {process_utils.py:190} INFO - Downloading threadpoolctl-3.2.0-py3-none-any.whl (15 kB)
[2023-10-11T04:52:21.984+0000] {process_utils.py:190} INFO - Installing collected packages: pytz, tzdata, threadpoolctl, six, numpy, joblib, scipy, python-dateutil, scikit-learn, pandas, lightgbm
[2023-10-11T04:52:28.744+0000] {process_utils.py:190} INFO - Successfully installed joblib-1.3.2 lightgbm-4.1.0 numpy-1.24.4 pandas-2.0.3 python-dateutil-2.8.2 pytz-2023.3.post1 scikit-learn-1.3.1 scipy-1.10.1 six-1.16.0 threadpoolctl-3.2.0 tzdata-2023.3
[2023-10-11T04:52:28.921+0000] {process_utils.py:190} INFO - 
[2023-10-11T04:52:28.921+0000] {process_utils.py:190} INFO - [notice] A new release of pip is available: 23.2 -> 23.2.1
[2023-10-11T04:52:28.921+0000] {process_utils.py:190} INFO - [notice] To update, run: python -m pip install --upgrade pip
[2023-10-11T04:52:29.012+0000] {process_utils.py:182} INFO - Executing cmd: /tmp/venvebjuz6_a/bin/python /tmp/venvebjuz6_a/script.py /tmp/venvebjuz6_a/script.in /tmp/venvebjuz6_a/script.out /tmp/venvebjuz6_a/string_args.txt /tmp/venvebjuz6_a/termination.log
[2023-10-11T04:52:29.025+0000] {process_utils.py:186} INFO - Output:
[2023-10-11T04:52:29.761+0000] {process_utils.py:190} INFO - Loading data...
[2023-10-11T04:52:29.762+0000] {process_utils.py:190} INFO - Traceback (most recent call last):
[2023-10-11T04:52:29.762+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/script.py", line 90, in <module>
[2023-10-11T04:52:29.763+0000] {process_utils.py:190} INFO -     res = train_task(*arg_dict["args"], **arg_dict["kwargs"])
[2023-10-11T04:52:29.763+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/script.py", line 40, in train_task
[2023-10-11T04:52:29.763+0000] {process_utils.py:190} INFO -     df_test = pd.read_csv(
[2023-10-11T04:52:29.763+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 912, in read_csv
[2023-10-11T04:52:29.763+0000] {process_utils.py:190} INFO -     return _read(filepath_or_buffer, kwds)
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 577, in _read
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -     parser = TextFileReader(filepath_or_buffer, **kwds)
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1407, in __init__
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -     self._engine = self._make_engine(f, self.engine)
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/lib/python3.8/site-packages/pandas/io/parsers/readers.py", line 1661, in _make_engine
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -     self.handles = get_handle(
[2023-10-11T04:52:29.764+0000] {process_utils.py:190} INFO -   File "/tmp/venvebjuz6_a/lib/python3.8/site-packages/pandas/io/common.py", line 859, in get_handle
[2023-10-11T04:52:29.765+0000] {process_utils.py:190} INFO -     handle = open(
[2023-10-11T04:52:29.765+0000] {process_utils.py:190} INFO - FileNotFoundError: [Errno 2] No such file or directory: '/opt/***/data/regression/regression.test'
[2023-10-11T04:52:30.158+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 221, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 395, in execute
    return super().execute(context=serializable_context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 602, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path, tmp_path)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 463, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
[Errno 2] No such file or directory: '/opt/***/data/regression/regression.test'
[2023-10-11T04:52:30.160+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=docker, task_id=virtualenv_python, execution_date=20231011T045205, start_date=20231011T045207, end_date=20231011T045230
[2023-10-11T04:52:30.174+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 1422 for task virtualenv_python (Process returned non-zero exit status 1.
[Errno 2] No such file or directory: '/opt/***/data/regression/regression.test'; 236)
[2023-10-11T04:52:30.203+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-10-11T04:52:30.213+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
